charts/kured/templates/daemonset.yaml

```diff
@@ -104,6 +104,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            {{- if .Values.extraEnvVars }}
```

Comment:
@audunsolemdal Could you please add a "end" for the "if".

Reply:
I believe this has to be left adjusted when you use indent. Also see nindent